app/services/hosts.ts

```diff
@@ -54,3 +55,15 @@ export class HostsService extends Service {
     return 'r2d2.streamlabs.com';
   }
 }
+
+export class UrlService extends Service {
+  @Inject('HostsService') private hosts: HostsService;
+
+  get protocol() {
+    return Util.useLocalHost() ? 'http://' : 'https://';
+  }
+
+  getStreamlabsApi(endpoint: string) {
+    return `${this.protocol}${this.hosts.streamlabs}/api/v5/slobs/${endpoint}`;
+  }
+}
```

Comment:
i'm not really sure why this exists

Reply:
It's for getting API endpoints for the case when you have the backend on your local PC

Reply:
For example it's hard to use `https` when you have a localy installed backend